script/before_install.sh

```diff
@@ -149,20 +149,17 @@ then
             exit 1
         fi
     fi
+fi
 
-    if [ "${ALPAKA_CI_STDLIB}" == "libstdc++" ]
+if [ ! -z "${ALPAKA_CI_CLANG_VER+x}" ]
+then
+    if [ "${ALPAKA_CI_CLANG_VER}" == 5 ]
```

Comment:
Do we still support clang-5? I think this check can be removed full.

Reply:
Unfortunately, we still have clang-5.

Reply:
Ok no problem, I thought clang-5 is not supporting C++17.